src/views/pages/Analytics/index.vue

```diff
@@ -0,0 +1,243 @@
+<template>
+  <div>
+    <!-- Stats Section -->
+    <div class="md-layout stats md-gutter">
+        <md-card class="md-layout-item md-elevation-6">
+          <md-card-header>
+            <md-card-header-text>
+              <div class="stat-heading">Milk Collected (Month)</div>
+              <div class="stat">{{collected}} L</div>
+            </md-card-header-text>
+          </md-card-header>
+        </md-card>
+        <md-card class="md-layout-item md-elevation-6">
+          <md-card-header>
+            <md-card-header-text>
+              <div class="stat-heading">Milk Delivered (Month)</div>
+              <div class="stat">{{delivered}} L</div>
+            </md-card-header-text>
+          </md-card-header>
+        </md-card>
+        <md-card class="md-layout-item md-elevation-6">
+          <md-card-header>
+            <md-card-header-text>
+              <div class="stat-heading">Loans Dispensed (Month)</div>
+              <div class="stat">{{disperesed}} UGX</div>
+            </md-card-header-text>
+          </md-card-header>
+        </md-card>
+    </div>
+    <!-- Graphs -->
+    <md-card class='md-elevation-10 graph'>
+      <graph 
+        v-bind:values="productTranscations" 
+        title="Milk-Collections-and-Deliveries" 
+        xUnits="Date" 
+        yUnits="Litres"
+      />
+    </md-card>
+    <md-card class='md-elevation-10 graph'>
+      <graph 
+        v-bind:values="moneyTranscations" 
+        title="Loan-Dispersals-and-Payments" 
+        xUnits="Date" 
+        yUnits="UGX"
+      />
+    </md-card>
+  </div>
+</template>
+
+<script>
+import graph from "../../components/AnalyticsGraph";
+import moment from "moment";
+import * as d3 from "d3";
+
+export default {
+  name: "analytics",
+  data() {
+    return {
+      productTranscations: [],
+      moneyTranscations: [],
+      collected: 0,
+      delivered: 0,
+      dispensed: 0,
+    };
+  },
+  components: {
+    graph,
+  },
+
+  created() {
+  // generate data 
+    this.productTranscations.push(this.fillCollections ());
+    this.productTranscations.push(this.fillExports ());
+    this.moneyTranscations.push(this.fillLoans ());
+    this.moneyTranscations.push(this.fillPayments ());
+
+  // calculate the stats
+    this.collected = this.calculateCollected ();
+    this.delivered = this.calculateDelivered ();
+    this.disperesed = this.calculateLoans ();
+  },
+  methods: {
+    fillCollections () {
```

Comment:
for these methods that put in test data, maybe rename the functions to like "generateTestCollections()" so that it is clear that these functions are not using real data

Reply:
even better, if we could just have a file that has functions for generating test/fake data and this file could easily call it so then when these are actually implemented and hooked up, it'll be an easy switch